### ISSUE TITLE
I am receiving the same error as @MounicaMercy

### ISSUE THREAD

**Issue by derek-olson** [nodejs/help#3009]
Hello,
I installed node version- 12.18.4 and when I'm trying to run gulp serve, this is the error im getting.
ReferenceError: primordials is not defined
    at fs.js:36:5
    at req_ (C:\Users\mounika.pasupunuri\hello-webpart\node_modules\natives\index.js:143:24)
    at Object.req [as require] (C:\Users\mounika.pasupunuri\hello-webpart\node_modules\natives\index.js:55:10)
    at Object.<anonymous> (C:\Users\mounika.pasupunuri\hello-webpart\node_modules\gulp\node_modules\graceful-fs\fs.js:1:37)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19)
    at require (internal/modules/cjs/helpers.js:72:18)
    at Object.<anonymous> (C:\Users\mounika.pasupunuri\hello-webpart\node_modules\gulp\node_modules\graceful-fs\graceful-fs.js:3:27)
    at Module._compile (internal/modules/cjs/loader.js:1137:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1157:10)
    at Module.load (internal/modules/cjs/loader.js:985:32)
    at Function.Module._load (internal/modules/cjs/loader.js:878:14)
    at Module.require (internal/modules/cjs/loader.js:1025:19)

Thank you in advance.

_Originally posted by @MounicaMercy in https://github.com/nodejs/help/issues/2613#issuecomment-703476716_

**Comment by MounicaMercy** [comment]
Withot helping or commenting you closed the issue?

**Comment by derek-olson** [comment]
@MounicaMercy, my apologies. Here is what happened and how I fixed it. 

I was using gulp ^4.0 and Node 10.15.1 I had everything running successfully in August, which is when I updated from gulp 3 to 4. 

I went to make some changes and received the error that you are. At that point I tried doing a number of things to fix the issue, most of which involved deleting node modules and reinstalling node. When I did this, I didn't realize that I deleted my global install of gulp, embarrassing. 

So after trying a million things what worked for me was:

  "scripts": {
    "gulp": "gulp",
    "preinstall": "npx npm-force-resolutions"
  },
  "resolutions": {
    "graceful-fs": "^4.2.4"
  },

I reinstalled gulp globally, and I also upgraded to Node 12.18.4 in the process. 

Hope that all helps.
